
try
{
    //例外発生の可能性のある処理を記述
}
catch(例外) when(条件)
{
    //例外が発生した場合の処理を記述
}

